src/components/ProductTours/TourCollapsible.tsx

```diff
@@ -2,28 +2,42 @@ import React, { FC, useState } from 'react';
 import { connect } from 'react-redux';
 
 import {
-  IconButton, Icon, OverlayTrigger, Tooltip,
+  IconButton, Icon, OverlayTrigger, Tooltip, Stack,
 } from '@openedx/paragon';
-import { Question } from '@openedx/paragon/icons';
+import { Question, TrendingUp } from '@openedx/paragon/icons';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
 import FloatingCollapsible from '../FloatingCollapsible';
 import messages from './messages';
 import { dismissOnboardingTour, reopenOnboardingTour } from '../../data/actions/enterpriseCustomerAdmin';
+import { Step } from './AdminOnboardingTours/OnboardingSteps';
+import { ADMIN_TOUR_TARGETS } from './AdminOnboardingTours/constants';
 
 interface Props {
   onboardingTourCompleted: boolean;
   onboardingTourDismissed: boolean;
   dismissOnboardingTour: () => void;
   reopenOnboardingTour: () => void;
+  onTourSelect?: (targetId: string) => void;
 }
 
+const QUICK_START_GUIDE_STEPS = [
+  {
+    icon: TrendingUp,
+    title: 'Track Learner Progress',
```

Comment:
The title should use the variable from the new messages file instead of being a static string